content/javascript/concepts/arrays/terms/slice/slice.md

```diff
@@ -1,45 +1,43 @@
 ---
 Title: '.slice()'
-Description: 'Returns a shallow copy of part of array, while original array is not modified.'
+Description: 'Creates a partial copy of an array without altering the original array.'
 Subjects:
   - 'Web Development'
   - 'Computer Science'
 Tags:
   - 'Arrays'
   - 'Methods'
+  - 'Elements'
 CatalogContent:
   - 'introduction-to-javascript'
   - 'paths/front-end-engineer-career-path'
 ---
 
-The **`.slice()`** array method returns a shallow copy of all or part of an array without modifying the original. A shallow copy duplicates the contents of an object into a new instance by reference; which is why changes to the copy are not reflected in the original.
+The **`.slice()`** [method](https://www.codecademy.com/resources/docs/javascript/methods) in JavaScript is utilized to select a segment of an [array](https://www.codecademy.com/resources/docs/javascript/arrays) and generate a new array from that selection. Importantly, this method creates this partial copy of an array, otherwise known as a _shallow copy_, without altering the original array.
```

Comment:
```suggestion
The **`.slice()`** [method](https://www.codecademy.com/resources/docs/javascript/methods) in JavaScript returns a partial copy of an array, otherwise known as a shallow copy, without altering the original array.
```

I have suggested to remove some redundant sentences. Also note that linking the linking array doesn't make sense as this entry will anyway be inside of the array concept.